v3/src/components/tool-shelf/tool-shelf.tsx

```diff
@@ -76,7 +76,8 @@ export const ToolShelf = observer(function ToolShelf({ document }: IProps) {
     {
       icon: <OptionsIcon className="icon-options"/>,
       label: t("DG.ToolButtonData.optionMenu.title"),
-      hint: t("DG.ToolButtonData.optionMenu.toolTip")
+      hint: t("DG.ToolButtonData.optionMenu.toolTip"),
```

Comment:
In V2 the tooltip is "Display a website, configure guide…" But here I'm just getting "Options"